url.bs

```diff
@@ -3172,23 +3180,26 @@ work.
         "publisher": "Unicode Consortium"
     },
     "UTS36": {
-      "href": "http://unicode.org/reports/tr36/",
-      "authors" : ["Mark Davis", "Michel Suignard"],
-      "title": "Unicode Security Considerations",
-      "publisher" : "Unicode Consortium"
+        "href": "http://www.unicode.org/reports/tr36/",
+        "authors": ["Mark Davis", "Michel Suignard"],
+        "title": "Unicode Security Considerations",
+        "publisher": "Unicode Consortium"
     }
 }
 </pre>
 
 <pre class=anchors>
-urlPrefix: https://w3c.github.io/FileAPI/; type: dfn
-    text: blob url store; url: #BlobURLStore
-urlPrefix: https://w3c.github.io/media-source/#idl-def-; type: interface
-    text: MediaSource
-urlPrefix: https://w3c.github.io/mediacapture-main/#idl-def-; type: interface
-    text: MediaStream
-url: http://www.unicode.org/reports/tr46/#ToASCII; type: dfn; text: toascii; spec: IDNA
-url: http://www.unicode.org/reports/tr46/#ToUnicode; type: dfn; text: tounicode; spec: IDNA
+# XXX Cannot use spec attribute here, as a Bikeshed bug currently prevents it
+# from coalescing autolinked and manually linked FILEAPI entries in the index.
```

Comment:
To elaborate, if this patch is applied:

```diff
diff --git a/url.bs b/url.bs
index 6ed22f1..a1e8d5f 100644
--- a/url.bs
+++ b/url.bs
@@ -1303,10 +1303,8 @@ <h3 id=url-parsing>URL parsing</h3>
  <li><p>If <var>url</var>'s <a for=url>scheme</a> is not
  "<code>blob</code>", return <var>url</var>.
 
- <!-- XXX spec attribute needed to work around a Bikeshed bug. See comment in pre.anchors for more
-      info -->
  <li><p>If <var>url</var>'s <a for=url>path</a> <a for=list>is empty</a> or <var>url</var>'s
- <a for=url>path</a>[0] is not in the <a spec=FILEAPI>Blob URL Store</a>, then return <var>url</var>.
+ <a for=url>path</a>[0] is not in the <a>Blob URL Store</a>, then return <var>url</var>.
  [[!FILEAPI]]
 
  <li><p>Set <var>url</var>'s <a for=url>object</a> to the entry in the <a>Blob URL Store</a>
@@ -3189,9 +3187,7 @@ <h2 id=acknowledgments class=no-num>Acknowledgments</h2>
 </pre>
 
 <pre class=anchors>
-# XXX Cannot use spec attribute here, as a Bikeshed bug currently prevents it
-# from coalescing autolinked and manually linked FILEAPI entries in the index.
-urlPrefix: https://w3c.github.io/FileAPI/
+spec: FILEAPI; urlPrefix: https://w3c.github.io/FileAPI/
     type: dfn; text: Blob URL Store; url: #BlobURLStore
 spec: IDNA; urlPrefix: http://www.unicode.org/reports/tr46/
     type: abstract-op; text: ToASCII; url: #ToASCII
```

the HTML changes in the following fashion:

```diff
diff --git a/url-pr-299.html b/url.html
index 088f353..6b1cd18 100644
--- a/url-pr-299.html
+++ b/url.html
@@ -2605,10 +2605,14 @@ work. </p>
    <li>
     <a data-link-type="biblio">[FILEAPI]</a> defines the following terms:
     <ul>
-     <li><a href="https://w3c.github.io/FileAPI/#dfn-Blob">Blob</a>
      <li><a href="https://w3c.github.io/FileAPI/#BlobURLStore">blob url store</a>
     </ul>
    <li>
+    <a data-link-type="biblio">[FILEAPI]</a> defines the following terms:
+    <ul>
+     <li><a href="https://w3c.github.io/FileAPI/#dfn-Blob">Blob</a>
+    </ul>
+   <li>
     <a data-link-type="biblio">[HTML]</a> defines the following terms:
     <ul>
      <li><a href="https://html.spec.whatwg.org/multipage/comms.html#eventsource">EventSource</a>
```